app/scripts/controllers/edit/autoscaler.js

```diff
@@ -78,6 +78,7 @@ angular.module('openshiftConsole')
     var horizontalPodAutoscalerVersion = APIService.getPreferredVersion('horizontalpodautoscalers');
     var limitRangesVersion = APIService.getPreferredVersion('limitranges');
 
+
```

Comment:
Rolled this back to minmal changes.

Note that  as it stands now & like `master`, the `showCPURequestWarning` will not trigger w/o hitting a certain set of conditionals, though the cpu request warning DOES appear on the DC page.